tests/test_js.py

```diff
@@ -1577,3 +1577,22 @@ async def error_handler(e):
 
         with pytest.raises(BadBucketError):
             await js.key_value(bucket="TEST3")
+
+class ConsumerReplicasTest(SingleJetStreamServerTestCase):
+    @async_test
+    async def test_number_of_consumer_replicas(self):
+        nc = await nats.connect()
+
+        js = nc.jetstream()
+        await js.add_stream(name="TESTREPLICAS", subjects=["test.replicas"])
```

Comment:
Hi @wallyqs Is there a way to make this stream R3 here?
This would strengthen the test so the streams is R3 and consumer is R1 and we can be 100% sure the replication factor for consumer is not inherited from the stream.

Reply:
I'd like that too, would have to make the tests run in cluster mode and right now they only run in single node (no helper for that in the tests atm)